README.md

```diff
@@ -38,8 +38,9 @@ build of each project, including running tests and checks such as linting. Run w
 
 ### Playground
 
-To run the "playground" locally, `cd` into the `playground` directory, and run `npm start`.
-This will launch a local web server and output the URL to visit to the console.
+To run the "playground" locally, build the repository, then `cd` into the `playground` directory, and run `npm start`.
+If you only want to build the functionality necessary to run the playground, you can use `python .\build.py --wasm --npm --play`.
+This will launch a local web server and output the URL to the console; copy that URL and open it in a browser to use the playground.
```

Comment:
```suggestion
This will launch a local web server and output the URL to the console; copy that URL and open it in a browser to use the playground.
If you only want to build the functionality necessary to run the playground, you can use `python .\build.py --wasm --npm --play`.
```

Reply:
Switched the order of sentences because the "local web server URL" sentence was a bit confusing if it remained at the end.